python/ray/train/examples/tune_cifar_torch_pbt_example.py

```diff
@@ -88,11 +113,12 @@ def train_func(config):
     )
 
     with FileLock(".ray.lock"):
+        data_dir = config.get("data_dir", "~/data")
```

Comment:
can we use something like `os.path.expanduser("~/.ray.lock")` instead? Ideally tie it to the `data_dir`. If each worker runs in a separate dir, then they will not use the same lock files.